projects/python/simulation/agricultural-simulation-dataset-generator/readme.md

```diff
@@ -0,0 +1,59 @@
+# Agricultural Simulation Dataset Generator
+This folder contains source for:
+- Generating agriculatural simulation dataset using a Webots simulator environment.
+- A python script for loading, viewing and using the generated datasets.
+
+<p float="left">
+  <img src="./examples/UGV_RGB.jpg" width=300 />
+  <img src="./examples/UGV_RGB_SEGMENTED.jpg" width=300 />
+</p>
+<p float="left">
+  <img src="./examples/UAV_RGB.jpg" width=300 />
+  <img src="./examples/UAV_RGB_SEGMENTED.jpg" width=300 />
+</p>
+
+# How to use
```

Comment:
Can you also please some additional details regarding running the generation utility, e.g., a) what is the default output path, b) what is the default dataset and how much space it takes (approx), c) does the user need to do anything apart from loading the webots world in order to generate the dataset?